README.md

```diff
@@ -7,7 +7,7 @@ and provide you with a set of assertions you can use to verify your process beha
 
 ## Prerequisites
 
-* Java 21+ when running with an embedded engine (`zeebe-process-test-extension`)
+* Java 21+ (17+ for >=`8.3`) when running with an embedded engine (`zeebe-process-test-extension`)
```

Comment:
Thanks for the suggestion @jonathanlukas ❤️ 

The statement isn't entirely correct. It implies that 8.4 would also allow 17+.

How about:
```
(17+ for `8.3.x`)
```

Reply:
Thanks @korthout . I changed it to <=8.3.x , my bad...